Build Issue

```
./arfc-pres.tex:70: Undefined control sequence.
<argument> \@@magyar@captionfix

l.70 \begin{document}

./arfc-pres.tex:70:  ==> Fatal error occurred, no output PDF file produced!
Transcript written on arfc-pres.log.
make: *** [all-via-pdf] Error 1
```

This is being caused by the captions package. I'm not sure if you use it, but it won't build on my machine if this is included. Will submit fixing pr.

--- a/arfc-pres.tex
+++ b/arfc-pres.tex
@@ -24,7 +24,6 @@
 \usepackage{tikz}
 \usepackage{amsfonts}
 \usepackage{amsmath}
-\usepackage{caption}  % allows center figures caption
 \usepackage{xspace}
 \usepackage{notoccite}
 \usepackage{graphicx}

